Imported target "Ableton::Link" includes non-existent path

## Environment

* SuperCollider version: 3.11
* Operating system: Linux Mint (same as Ubuntu)
* Other details (Qt version, audio driver, etc.):
Qt 5.9.5
audio driver: Jack


## Steps to reproduce

After dowloading the 3.11 source files and i do:
cmake -L ..
in the build folder I get:

CMake Error in lang/CMakeLists.txt:
  Imported target "Ableton::Link" includes non-existent path

    "/home/marcello/Documents/SuperCollider-Source/external_libraries/link/cmake_include/../modules/asio-standalone/asio/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in lang/CMakeLists.txt:
  Imported target "Ableton::Link" includes non-existent path

    "/home/marcello/Documents/SuperCollider-Source/external_libraries/link/cmake_include/../modules/asio-standalone/asio/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in lang/CMakeLists.txt:
  Imported target "Ableton::Link" includes non-existent path

    "/home/marcello/Documents/SuperCollider-Source/external_libraries/link/cmake_include/../modules/asio-standalone/asio/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.



CMake Error in lang/CMakeLists.txt:
  Imported target "Ableton::Link" includes non-existent path

    "/home/marcello/Documents/SuperCollider-Source/external_libraries/link/cmake_include/../modules/asio-standalone/asio/include"

  in its INTERFACE_INCLUDE_DIRECTORIES.  Possible reasons include:

  * The path was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and references files it does not
  provide.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -228,6 +228,7 @@ endif()
 
 option(SC_SYMLINK_CLASSLIB "Place a symlink of SCCLassLibrary instead of copying" OFF)
 
+option(SYSTEM_ABLETON_LINK "Use link from system" OFF)
 option(SYSTEM_BOOST   "Use boost libraries from system" OFF)
 option(SYSTEM_YAMLCPP "Use yaml-cpp library from system" OFF)
 

--- a/lang/CMakeLists.txt
+++ b/lang/CMakeLists.txt
@@ -252,7 +252,12 @@ endif()
 
 if (SC_ABLETON_LINK)
 	message(STATUS "Compiling with Ableton Link support")
-	include(../external_libraries/link/AbletonLinkConfig.cmake)
+	if(SYSTEM_ABLETON_LINK)
+		find_package(AbletonLink NAMES AbletonLink ableton-link link REQUIRED)
+	else()
+		include(../external_libraries/link/AbletonLinkConfig.cmake)
+	endif()
+
 	target_link_libraries(libsclang Ableton::Link)
 
 	add_definitions(-DSC_ABLETON_LINK)

